app/_hub/kong-inc/ldap-auth-advanced/0.33-x.md

```diff
@@ -104,7 +104,7 @@ params:
       default: "`false`"
       value_in_examples:
       description: |
-        Whether consumer is optional
+        Whether consumer mapping is optional. If `consumer_optional=true`, then the plugin will not attempt to associate a consumer with the LDAP authenticated user. If `consumer_optional=false`, LDAP authenticated users can still access upstream resources. If you wish to prevent LDAP users who are not associated with consumers, then you must set `consumer_optional=false`, set the `anonymous` field to an existing consumer_id, and then use the Request Termination plugin to deny any requests from the anonymous consumer.
```

Comment:
```suggestion
        Whether consumer mapping is optional. If `consumer_optional=true`, the plugin will not attempt to associate a consumer with the LDAP authenticated user. If `consumer_optional=false`, LDAP authenticated users can still access upstream resources. To prevent access from LDAP users that are not associated with consumers, set `consumer_optional=false`, set the `anonymous` field to an existing `consumer_id`, then use the Request Termination plugin to deny any requests from the anonymous consumer.
```